mmv1/products/cloudrun/Service.yaml

```diff
@@ -830,6 +839,20 @@ properties:
                                   not specified, the volume defaultMode will be used. This might be in
                                   conflict with other options that affect the file mode, like fsGroup, and
                                   the result can be other mode bits set.
+                    - !ruby/object:Api::Type::NestedObject
+                      name: emptyDir
```

Comment:
Could you add a test for each of the new fields?

Reply:
These fields are covered by the added example.